### PR TITLE
feat: add support for gfx1100 devices

### PR DESCRIPTION
**Pull request by evshiron**
Thread ref: https://github.com/RadeonOpenCompute/ROCm/issues/1880

### CODE REVIEW

File: tensorflow/compiler/xla/stream_executor/device_description.h [hunk 1]
```diff
@@ -188,7 +189,7 @@ class RocmComputeCapability {
     return {"gfx908", "gfx90a"};
```
Comment (cromefire) [inline tensorflow/compiler/xla/stream_executor/device_description.h:189]: BF16 should also work for RDNA 3 according to [AMD's documentation](https://gpuopen.com/learn/wmma_on_rdna3/), so it should be fine to be enabled. It did work for me to add `gfx1100` there, but I'm not sure how to properly test that it's really fine (it didn't make any difference performance wise in my testing).

image

**Comment by jayfurmanek** [comment]
Hi, Thanks for the PR!
I'll review. There are likely a few more places we'll need to specify the new arch. 
It also may be a bit before we can merge as there are some needed miopen updates.

**Review by jayfurmanek — APPROVED** [review]
Thanks so much @evshiron!  I'll follow up with another PR for a missing piece in XLA and miopen.

**Comment by evshiron** [comment]
@jayfurmanek 

Greetings. Sorry to bother you, but the wheels from CI doesn't work as indicated here:

* https://github.com/ROCmSoftwarePlatform/tensorflow-upstream/issues/2142

And I would like to know when `tensorflow-rocm` will ship with Navi 31 support?

**Pull request merged**